editor.planx.uk/src/@planx/components/MapAndLabel/test/public.test.tsx

```diff
@@ -171,8 +171,6 @@ describe("validation and error handling", () => {
     await clickContinue(user);
 
     await checkErrorMessagesPopulated();
-
-    expect(tabOne).toHaveStyle("border-left: 5px solid #D4351C");
```

Comment:
This was failing - presumably as JSDom isn't handling media queries (which the style is now based on).